src/Stratis.Bitcoin.Features.Consensus/CoinViews/CachedCoinView.cs

```diff
@@ -85,6 +85,9 @@ private class CacheItem
         /// <summary>Coin view at one layer below this implementaiton.</summary>
         private readonly ICoinView inner;
 
+        /// <summary>Pending list of rewind data to be persisted to a persistent storage.</summary>
```

Comment:
Add a remarks section saying that access to this object has to be protected with lock object. See other fields on the class for similar remarks section.

Also expand lock objects comment to include this new contract

Reply:
done